src/less/menu/ds4/menu-base.less

```diff
@@ -145,7 +145,7 @@ svg.fake-menu__status {
 
 span.menu__icon, // .menu__icon deprecated in v3.2.0
 span.fake-menu__icon { // .fake-menu__icon deprecated in v3.2.0
-    background-image: url('data:image/svg+xml,%3Csvg%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20width%3D%2732%27%20height%3D%2732%27%3E%3Cpath%20fill%3D%22%230654ba%22%20d%3D%22M15.94%2025.294l-15.473-17.229h30.953l-7.742%208.616z%22%3E%3C/path%3E%3C/svg%3E');
+    background-image: url('data:image/svg+xml;base64,@{ds4-icon-arrow-down-base64}');
```

Comment:
@seangates The `menu__icon` was deprecated in v3.2.0 in favor of using the `expand-btn`, so these changes wont be reflected on the website. Although it is still used on the test page.

Reply:
We should change that, then.

Reply:
Change what? Not sure I follow.